pkg/parser/external/java_parser.go

```diff
@@ -34,56 +32,69 @@ func newJavaParser(typ string) *javaParser {
 	return &javaParser{typ: typ}
 }
 
-type parseResult struct {
-	Statements []string `json:"statements"`
-	Position   int      `json:"position"`
-	Error      string   `json:"error"`
+func getServerAddress() string {
+	ip := getEnv("SQLFLOW_EXTERNAL_PARSER_IP", "localhost")
+	port := getEnv("SQLFLOW_EXTERNAL_PARSER_PORT", "12345")
```

Comment:
12345 is a magic number.  Another use of this number is at https://github.com/sql-machine-learning/sqlflow/pull/1792/files#diff-f612dacf52e3192f6479782d99b95e1cR55. Let us either remove it or define a global constant. I think we can remove it.

Reply:
I will remove it.